packages/edit-site/src/components/block-editor/use-post-link-props.js

```diff
@@ -15,6 +15,10 @@ export function usePostLinkProps() {
 	const history = useHistory();
 
 	return ( params, state ) => {
-		return getPostLinkProps( history, params, state );
+		return getPostLinkProps(
```

Comment:
I believe `getPostLinkProps` is probably a function that shouldn't come from `link.js` file (which in my mind should't have "post" in it), maybe it's the name of the function that is wrong.

Reply:
Yeh, I think this ties in with the naming issue [mentioned here](https://github.com/WordPress/gutenberg/pull/57700#discussion_r1464407249). Are you happy to fix that in a follow-up rename PR?  What did you think of `getEntityNavigation` ?

Reply:
I think it should move to the current file rather than stay in `link` basically, regardless of its name. That said, that's very very minor.